src/Listbox.js

```diff
@@ -314,7 +322,15 @@ export const Listbox = {
     },
     open() {
       this.$data.isOpen.value = true
-      this.focus(this.$props.value)
+      // Multi selects focus the first selected option when opened
```

Comment:
Is this a W3 standard? If not, that's cool. If it is, I'd love to see that spec.

Reply:
Yes, it is listed in the [WAI-ARIA practices for Listboxes](https://www.w3.org/TR/wai-aria-practices/#listbox_kbd_interaction).

Reply:
Can you include that link in the code comment?